src/domUpdates.js

```diff
@@ -4,26 +4,41 @@ import Destination from '../src/destination.js'
 
 let domUpdates = {
 
+  travelers: [{}],
   traveler: {},
-  trip: {},
+  trips: {},
   destinations: [{}],
 
   getTravelData: () => {
     Promise.all([
-      fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/48'),
+      fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers'),
       fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips'),
       fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/destinations/destinations')
     ])
       .then(responses => Promise.all(responses.map(response => response.json())))
       .then(responses => {
-        domUpdates.traveler = responses[0].travelers;
-        domUpdates.trip = responses[1].trip;
+        domUpdates.travelers = responses[0].travelers;
+        domUpdates.trips = responses[1].trips;
         domUpdates.destinations = responses[2].destinations;
+        domUpdates.getTraveler();
         domUpdates.getDestinations();
+        console.log(domUpdates.traveler);
       })
       .catch(error => console.log(error))
+  },
 
+  getTraveler: () => {
+    domUpdates.traveler = new Traveler(domUpdates.travelers[47], domUpdates.trips, domUpdates.destinations)
```

Comment:
I lied. This is separation of DOM/data model. It's just that your data is in your DOM file. Which I think is okay. It might be cleaner if the new Traveler was instantiated in the `index.js` file, and then ONLY the information (say, traveler.name) that domUpdates needs is passed in like (from the index.js file) `domUpdates.welcomeUser(traveler.name)`

Reply:
If you have time today, can we have a quick call about this? There's some back and forth in your comments here and I want to make sure I understand what you're saying. It's also possible that it's early and I need more coffee.

Reply:
So, I wonder if part of the confusion here is that I haven't done my login page yet. Ultimately, the displayed traveler will be instantiated based on login/travelerData, but for now, so I can see stuff on my page, I'm passing in a specific user. Does that make more sense?